queries.sql

```diff
@@ -7,4 +7,37 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered IS TRUE;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT (id, name, species) from animals;
+ROLLBACK;
+SELECT (id, name, species) from animals;
+
+BEGIN;
+UPDATE animals SET species = 'digmon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] After deleting from the animals table you should select all from the animal table to verify if the change persists after deleted 🙃 🙃

Reply:
Thank you I have selected all from the animal table. 😁